### ISSUE TITLE
Hi @qazbnm456,do you have meet this problem?

### ISSUE THREAD

**Issue by xzycn** [LulumiProject/lulumi-browser#38]
Hi, do you have meet this [problem](https://github.com/electron/electron/issues/11471)?
Just as MarshallOfSound said: 

> window.opener support doesn't quite work in Electron primarily due to our process model, this may change at some point in the future.

The underlying issue confused me very much :(

**Comment by xzycn** [comment]
I have tested with **lumi-browser** in some situations,trouble with the same issue. :( :(

**Comment by qazbnm456** [comment]
Yes,  I've been suffering the pain of `window.opener` related bugs for a really long time. Hope they will fix the issue soon. 😞

**Comment by xzycn** [comment]
@qazbnm456 I have to stop my project that has cast my a lot time, it's a disaster :(
Maybe we can try nwjs?But,almost all things we have done before need to be redone.

**Comment by qazbnm456** [comment]
For some personal preferences, I'll still stick to Electron instead of NW.js actually. Maybe you can give it a try, but lots of things indeed need to be reimplemented. 😕

**Comment by xzycn** [comment]
@qazbnm456  Hi, Electron 2.0.0 has been released yesterday :), but the issue still :(
Do you find a way out now?

**Comment by qazbnm456** [comment]
Nop, I think the issues will only be resolved when the upstream lands patches or something. However, the progress is slow and priority is not that high currently. 😭

**Comment by xzycn** [comment]
@qazbnm456 Hi, sorry for my delaying reply.I created a [gist](https://gist.github.com/xzycn/aa21d2157fd3f09295743ca491b6f644) for **a minimal repro**, you could check it in your free time :)
Note: You should check the **window.opener** in console within new window.

**Comment by qazbnm456** [comment]
@xzycn Thanks! I'll notify you once I get some time and make some progress.

**Comment by xzycn** [comment]
@qazbnm456 Hi, I have made some changes that make it more reasonable  in the gist.
However, there is a weird thing: the param **webPreferences .nativeWindowOpen**  is noneffective when mainWindow initial.But putting it in inline of webview tag is effective.like this:
`<webview src="wv.html" webpreferences="nativeWindowOpen=yes" allowpopups></webview>`

**Comment by qazbnm456** [comment]
@xzycn Is that another problem? Could you please open another issue that addresses and describes the details of the problem? Thanks!

**Comment by xzycn** [comment]
@qazbnm456 sorry for my hehavior, it actually is a problem i met in my project not in lulumi ORZ.

**Comment by xzycn** [comment]
@qazbnm456 Hi,my friend.I got a [solution](https://github.com/brave/muon), it's a fork of electron ,maybe you could try,it has a feature:

> scriptable window.opener support

:) enjoy.
